service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/DraftCMOControllerTest.java

```diff
@@ -86,14 +95,24 @@ void aboutToSubmitShouldPopulateHiddenHearingDateField() throws Exception {
                     .label(date.plusDays(5).toString())
                     .build());
 
-        Map<String, Object> data = ImmutableMap.of("cmoHearingDateList", dynamicHearingDates);
+        Map<String, Object> data = ImmutableMap.of("cmoHearingDateList", dynamicHearingDates,
+            "allParties", createDirection(null));
 
         AboutToStartOrSubmitCallbackResponse callbackResponse = getResponse(data, "about-to-submit");
 
         assertThat(callbackResponse.getData()).doesNotContainKey("cmoHearingDateList");
 
         CaseData caseData = mapper.convertValue(callbackResponse.getData(), CaseData.class);
+        CaseManagementOrder caseManagementOrder = caseData.getCaseManagementOrder();
+
+        assertThat(caseManagementOrder).extracting("id", "hearingDate")
+            .containsExactly(fromString("b15eb00f-e151-47f2-8e5f-374cc6fc2657"), date.plusDays(5).toString());
```

Comment:
Aren't we already checking that couple lines below?

Reply:
Ah missed that! Yep duplicated assertion, will remove.